docs/mobility/slam/cartographer/_index.md

```diff
@@ -74,49 +76,101 @@ See Viam's [Pricing](https://www.viam.com/product/pricing) for more information.
 
   {{< /alert >}}
 
+#### Movement sensors (optional)
+
+You can use data from one or more movement sensors on your machine to supplement the required LiDAR data.
+If you choose to use movement sensor data for SLAM, you can:
+
+- Add only inertial measurement unit (IMU) data
+  - Requires a movement sensor that supports [`AngularVelocity`](/mobility/navigation/#angular-velocity) and [`LinearAcceleration`](/mobility/navigation/#linear-acceleration) readings
+- Add only odometry data
+  - Requires a movement sensor that collects [`Position`](/mobility/navigation/#position) and [`Orientation`](/mobility/navigation/#orientation) data (for example, [`wheeled-odometry`](/components/movement-sensor/wheeled-odometry/))
+- Add both IMU _and_ odometry data
+  - Requires all four of the above kinds of data, merged together using the [`merged` movement sensor model](/components/movement-sensor/merged/)
+- If you choose this option, be sure to configure data capture on the `merged` sensor and not on the individual movement sensors when following the steps below.
+
 ### Create a new map
 
 To create a new map, follow the instructions below.
 Creating a new map uses an instance of the cartographer module running in the cloud.
 
-1. Enable data capture and configure your `cartographer` SLAM service
+{{< tabs name="Create new map">}}
+{{% tab name="Config Builder" %}}
 
-   Follow the steps below to enable data capture and add the `cartographer` module to your machine:
+1. Add the data management service to your machine:
 
-   {{< tabs name="Create new map">}}
-   {{% tab name="Config Builder" %}}
+   Navigate to the **Services** subtab on your machine's **Config** tab.
+
+   Click **Create service** in the lower-left corner of the page.
+   Choose `Data Management` as the type and specify a name for your data management service, for example `Data-Management-Service`.
+   Click **Create**.
+
+   On the panel that appears, you can manage the capturing and syncing functions.
+   You can also specify the **directory**, the sync **interval**, and any **tags** to apply to captured data.
+   See the [data management service](/data/) for more information.
+
+2. Enable data capture for your camera, and for your movement sensor if you would like to use IMU data, odometry data, or both:
+
+   Find the component's card on your machine's **Config** tab.
+   Click `Add Method`, then specify the method type and the capture frequency.
+
+   - For the required LiDAR camera, choose the `NextPointCloud` method.
+     Set the capture frequency.
+     `5` hertz is a good starting place for most applications.
+
+     {{<imgproc src="/mobility/slam/rplidar-capture.png" resize="x1100" declaredimensions=true alt="An R P lidar camera configured in the Viam app config builder with next point cloud configured for capture at 5 hertz." >}}
+
+   - To capture data from one or more movement sensors:
+
+{{< tabs name="Movement sensor options" >}}
```

Comment:
This section renders fully left-aligned, rather than as substeps under `2` --> `To capture data from one or more movement sensors:` Looks like if the tabset (IMU, ODO, BOTH) and the tip (Note that data capture ...) would need to be bumped rightward.

Everything else here looks good!

Reply:
Try my suggestion - I assume you may have indented the content as well as the shortcode rather than just the shortcode

Reply:
Per offline debugging attempts and convo, it may not be possible to get this to indent in the way we want to. I don't know that we have any examples in our docs of indented tabs that are within lists within outer tabs. Leaving as-is.